parallel_gamit/pyNetwork.py

```diff
@@ -193,8 +194,9 @@ def __init__(self, cnn, archive, GamitConfig, stations, date,
 
     def make_clusters(self, points, stations, net_limit=NET_LIMIT):
         # Run initial clustering using bisecting 'q-means'
-        qmean = BisectingQMeans(n_clusters=2, init='random', n_init=50,
-                                algorithm='lloyd', max_iter=8000)
+        qmean = BisectingQMeans(min_clust_size=4, opt_clust_size=20,
```

Comment:
@demiangomez answering you question on slack-- the quickest thing to try is to change the `min_clust_size` parameter from "4" to "2". Right now qmeans is refusing to execute any cluster splits that result in a child cluster with membership of less than 4. Setting this to 2 will mean that the only cluster size that **isn't** allowed is a single lone station. Because this is pre `overcluster` and the cluster membership will grow to overlap neighboring subnetworks, I'd expect some population of network size around >5 and >10 bins, where the current parameters setting has a *smallest* cluster size of 22. You could also drop `opt_clust_size` if you want to move the mode of the distribution down from 34...